CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 * Validation of component slug ([#153])
+* `--interactive` option to prompt push confirmation 
```

Comment:
```suggestion
* `--interactive` option to prompt push confirmation ([#157])
```

And please add the corresponding link to the bottom of this page:
`[#157]: https://github.com/projectsyn/commodore/pull/157`